src/CovidAppointmentTable.js

```diff
@@ -39,9 +39,13 @@ export function sortAndFilterData(
     { sortKey, sortAsc },
     onlyShowAvailable
 ) {
+    const tooStaleMinutes = 60;
+    const oldestGoodTimestamp = new Date() - (tooStaleMinutes * 60 * 1000);
+
     const filteredData = onlyShowAvailable
```

Comment:
I think it would be more readable if you chained `filter`s, so maybe like
```
let filteredData = data.filter((entry) => (entry.timestamp >= oldestGoodTimestamp))
if (onlyShowAvailable) {
    filteredData = filteredData.filter((entry) => entry.hasAppointments);
}
```

Reply:
Agreed. Also found logic error for entries with ZERO timestamp.